OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -64,6 +64,12 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 			if (!attack.Info.IgnoresVisibility && type == TargetType.Actor && !Target.Actor.HasTrait<FrozenUnderFog>() && !self.Owner.CanTargetActor(Target.Actor))
 				return NextActivity;
 
+			// Drop the target if it is being captured by our own team.
+			var capturable = Target.Actor.TraitOrDefault<Capturable>();
+			var capturesinfo = Target.Actor.TraitOrDefault<CapturesInfo>();
```

Comment:
`Captures` would be on the engineer or whatever, not on the building, so checking for `CapturesInfo` won't give you anything (and I don't think you can check for sabotage with the current setup (unless we add a reference inside `Capturable` to the actor that is performing the capturing)).